setup.py

```diff
@@ -23,80 +26,97 @@
 pyplinkio_include_dir = pyplinkio_src_dir
 pyplinkio_src_files = glob.glob(os.path.join(pyplinkio_src_dir, "*.c"))
 
-cplinkio = Extension(
-    "plinkio.cplinkio",
-    libplinkio_src_files + libcsv_src_files + pyplinkio_src_files,
-    library_dirs=[],
-    include_dirs=[libplinkio_include_dir, libcsv_include_dir, pyplinkio_include_dir],
-    libraries=[],
-    language="c",
-    extra_compile_args=[],
-    define_macros=[],
-)
+with tempfile.TemporaryDirectory() as tmp_include_dir:
```

Comment:
This part is a bit errorprone if the snp_lookup.h.in is changed. Perhaps we can pass PLINKIO_BYTE_ORDER as a compiler option instead. Like `-DPLINKIO_BYTE_ORDER=<ENDIAN>` that way we could just pass that to define_macros or extra_compile_args in the python Extension. We could have a 
```c
#ifndef PLINKIO_BYTE_ORDER
#cmakedefine PLINKIO_BYTE_ORDER @PLINKIO_BYTE_ORDER@
#endif
```

to allow the c-library to work correctly. Not sure what the best approach is.

Reply:
I added "#ifndef PLINKIO_BYTE_ORDER" and made the file handling in setup.py robust.

Reply:
I refactored the file handling code. (https://github.com/mfranberg/libplinkio/pull/34/commits/06a58be1f4f5353ffd0c5e0f4d4b9770c4cdf903)